code/game/objects/items/weapons/grenades/chem_grenade.dm

```diff
@@ -106,7 +106,7 @@
 				var/temp_reagents = new/list()
 				var/reagents_text = ""
 				for(var/obj/item/weapon/reagent_containers/G in beakers)
-					if(istype(G, /obj/item/weapon/reagent_containers/glass/beaker) || istype(G, /obj/item/weapon/reagent_containers/glass/bottle) || istype(G, /obj/item/weapon/reagent_containers/food/drinks))
+					if(istype(G, /obj/item/weapon/reagent_containers/glass/beaker))
```

Comment:
this removes adminlogging

Reply:
Huh, I forgot to change this part around.

Reply:
Solved (TM)